src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -7,7 +7,6 @@
 import javafx.collections.ObservableList;
 
 import seedu.address.model.assignment.*;
```

Comment:
import seedu.address.model.assignment.*;  such * should be replaced with specific import later

Reply:
Good catch, I have made changes to this